src/test/resources/application-test.yaml

```diff
@@ -0,0 +1,14 @@
+server:
+  port: 5023
+
+network-store-server:
+  base-uri: http://localhost:8080
+
+actions-server:
+  base-uri: http://localhost:5021
```

Comment:
port 5021 is already assigned to cgmes boundary server

Reply:
Removed